test_scripts/Defects/8_0/3788/002_allow_RC_permission_for_App_after_PTU_with_REMOTE_CONTROL_appHMIType.lua

```diff
@@ -0,0 +1,46 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3788
+---------------------------------------------------------------------------------------------------
+-- Description: Check SDL assigned RC permissions for the App after PTU with 'REMOTE_CONTROL' appHMIType
+-- in case App is registered with 'MEDIA' appHMIType
```

Comment:
I would propose to rewrite description in a following way:
> Check SDL assigns RC permissions for the App in case:
> - Initially app has registered with 'MEDIA' appHMIType
> - appHMIType is changed to 'REMOTE_CONTROL' within PTU